### ISSUE TITLE
Service Dashboard uses incorrect tags

### ISSUE THREAD

**Issue by agregory999** [wavefrontHQ/wavefront-proxy#539]
When using preprocessor rules to change tags (like cluster and shard), the service dashboard does not respect the changes.  The traces show the modified tags and they are filterable on the main application status page, but you must choose "none" on the service dashboard if the jaeger app didn't set "cluster" or "shard".  Even if the preprocessor rules change it, the dropdowns are broken.

**Comment by sushantdewan123** [comment]
Did you use preprocessor rule to just change span tags in the Jaeger integration?

**Comment by agregory999** [comment]
Yes I used the preprocessor rules.  In my case, the acme-fitness app is not including cluster or shard tags, which the Service Dashboard wants to use for filtering.  In fact, they get set to "none", or at least from the preprocessor's perspective.  When I tried adding the tags, I actually got my new tag -and- "none".  So I used the following:

  preprocessor:
    rules.yaml: |
      '30001':
      - rule    : drop-cluster-none
        action  : spanDropTag
        key     : cluster
      - rule    : drop-shard-none
        action  : spanDropTag
        key     : shard
      - rule    : cluster-span-tag
        action  : spanAddTag
        key     : cluster
        value   : gregoryan-tkg-wlc
      - rule    : cluster-shard-tag
        action  : spanAddTag
        key     : shard
        value   : development

The Trace page shows my correctly (deleted and re-added) tags.
